### PR TITLE
Added pyramidal family morphology and subclass

### LINKED ISSUE #376: NTR: pyramidal family morphology
**Term:** 
pyramidal family morphology

**Definition:**
A cell morphology that inheres in neurons which have a pyramidal shaped soma, a single axon, a large apical dendrite and multiple basal dendrites.
doi:10.1016/b978-0-12-369497-3.10004-4

**Parent Class:**
Cell Morphology #364

**Others:**
Pyramidal (Shape): SAO:1465673360, ILX:0109553

**Notes:**
Related to https://github.com/obophenotype/cell-ontology/issues/751

### LINKED ISSUE #377: NTR: tufted pyramidal morphology
**Term:** 
tufted pyramidal morphology

**Definition:**
A pyramidal cell morphology that inheres in neurons which have a distinct tuft formation, distal from the soma.
PMID:30715238

**Parent Class:**
pyramidal morphology #376 

**Other Notes:**
ilxtr:TuftedPyramidalPhenotype

### LINKED ISSUE #378: NTR: untufted pyramidal morphology
**Term:** 
untufted pyramidal morphology

**Definition:**
A pyramidal cell morphology that inheres in neurons which lack a clear tuft formation but extend to large radial distances.
PMID:30715238

**Parent Class:**
pyramidal morphology #376 

**Other Notes:**
Neocortex Untufted Pyramidal Cell:  ILX:0381350  
ilxtr:UntuftedPyramidalPhenotype

### LINKED ISSUE #379: NTR: stellate pyramidal morphology
**Term:** 
stellate pyramidal morphology

**Definition:**
A pyramidal cell morphology that inheres in neurons which lack a tuft formation but extend to small radial distances forming a star-like shape.
PMID:30715238

**Parent Class:**
pyramidal morphology #376 

**Other Notes:**
ilxtr:StarPyramidalPhenotype

### LINKED ISSUE #380: NTR: inverted pyramidal morphology
**Term:** 
inverted pyramidal morphology 

**Definition:**
A pyramidal cell morphology that inheres in neurons which have an apical tree which is oriented towards the white matter.
PMID:30715238

**Parent Class:**
pyramidal morphology #376 

**Other Notes:**
ilxtr:InvertedPyramidalPhenotype

### LINKED ISSUE #381: NTR: horizontal pyramidal morphology
**Term:** 
horizontal pyramidal morphology

**Definition:**
A pyramidal cell morphology that inheres in neurons which have an apical tree which is oriented parallel to the pia.
PMID:30715238

**Parent Class:**
pyramidal morphology #376 

**Other Notes:**
ilxtr:HorizontalPyramidalPhenotype

### LINKED ISSUE #395: NTR: standard pyramidal morphology
Term:
standard pyramidal morphology

Synonyms:
vertical pyramidal morphology
pyramidal morphology

Definition:
A pyramidal cell morphology that inheres in neurons which have a standard pyramidal shape with its apical dendrite aligned vertically. 

Parent Class:
pyramidal morphology #376

Others:
ilxtr:PyramidalPhenotype
disjoint with horizontal pyramidal morphology #381

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #376
Fixes #377
Fixes #378
Fixes #379
Fixes #380
Fixes #381
Fixes #395

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -21856,6 +21856,66 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
...
+xref: ilxtr:HorizontalPyramidalPhenotype
+is_a: PATO:0070015 ! pyramidal family morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070018
+name: tufted pyramidal morphology
+def: "A pyramidal cell morphology that inheres in neurons which have a distinct tuft formation, distal from the soma." [PMID:30715238]
+xref: ilxtr:TuftedPyramidalPhenotype
+is_a: PATO:0070015 ! pyramidal family morphology
```
Comment (patrick-lloyd-ray) [inline src/ontology/pato-edit.obo:21892]: Is this the correct hierarchy? I thought 'pyramidal family morphology' was supposed to be the superclass with 'standard pyramidal morphology' and 'horizontal pyramidal morphology' as disjoint subclasses (as you have it here, correctly, I think). But my understanding from Tom's comments were that 'tufted pyramidal morphology', 'untufted pyramidal morphology', 'stellate pyramidal morphology', and 'inverted pyramidal morphology' were subclasses of 'standard pyramidal morphology' not direct children of 'pyramidal family morphology', which is a helper class of sorts.  

Also (small issue) I think it's standard to have the parent term as the genus in the textual definition, but I don't see 'pyramidal cell morphology' as a parent term here. Maybe this is lending to my confusion. If this is confusing to you, I'm happy to add line comments for each change that you can accept or reject accordingly.

**Review by patrick-lloyd-ray — COMMENTED** [review]
In-line comments.

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/pato-edit.obo:21892]: Oh, sorry, totally misunderstood, though I wonder if there are horizontal pyramidal that are also considered tufted etc? Do need input on this. 

>Also (small issue) I think it's standard to have the parent term as the genus in the textual definition, but I don't see 'pyramidal cell morphology' as a parent term here.

oh yeah, that was me forgetting to change it to 'pyramidal family morphology' - but I guess it does bring up this disconnect in understanding. Will change accordingly once we get some input from @tgbugs, thanks for pointing it out @patrick-lloyd-ray 

>I'm happy to add line comments for each change that you can accept or reject accordingly.

no need for this, will make all the changes once we figure out whats the best way to approach :) thanks!

**Review by dosumis — APPROVED** [review]

**Comment by tgbugs** [comment]
All looks good to me.

**Review by tgbugs — APPROVED** [review]

**Pull request merged**